src/coffee/directives/api/map.coffee

```diff
@@ -36,7 +36,7 @@ angular.module('uiGmapgoogle-maps.directives.api')
         transclude: true
         replace: false
         #priority: 100,
-        template: '<div class="angular-google-map"><div class="angular-google-map-container"></div><div ng-transclude style="display: none"></div></div>'
+        template: '<div class="angular-google-map"><div class="angular-google-map-container" id="angular-google-map-container"></div><div ng-transclude style="display: none"></div></div>'
```

Comment:
I didn't think about the state where there are multiple maps. Thanks for the info. GEE calls getElementById on the first parameter, so you have to pass it an ID string. I'll modify so that each map container gets a unique ID.